File: 01.ProgrammingFundamentals/C#/02.ConditionalStatementsAndLoops/Lab/P12_NumberChecker/P12_NumberChecker.cs
using System;

namespace P12_NumberChecker
{
    class P12_NumberChecker
    {
        static void Main(string[] args)
        {
            try
            {
                int number = int.Parse(Console.ReadLine());
                Console.WriteLine("It is a number.");
            }
            catch (FormatException ex)
            {
                Console.WriteLine("Invalid input!");
            }
        }
    }
}

File: ProgrammingFundamentals/Java/02.ConditionalStatementsAndLoops/Lab/src/P12_NumberChecker.java
import java.util.Scanner;
public class P12_NumberChecker {
    public static void main(String[] args) {

        Scanner console = new Scanner(System.in);

        try {
            int number = Integer.parseInt(console.nextLine());
            System.out.println("It is a number.");
        } catch (NumberFormatException ex) {
            System.out.println("Invalid input!");
        }
    }
}
